.spellcheck.yml

```
@@ -0,0 +1,18 @@
+matrix:
+- name: Markdown
+  aspell:
+    lang: en
```

Will this work?

```suggestion
    lang: en_GB
```

The action's readme page says it supports British English.